tools/log-collector/log_collector.go

```diff
@@ -30,16 +32,33 @@ import (
 	"github.com/trilioData/tvk-plugins/internal"
 )
 
+type GroupVersionKind struct {
+	Group   string `json:"group"`
+	Version string `json:"version"`
```

Comment:
add `omitempty`
If `version` not provided, use serverPreferredVesion
If `group` not provided, use "core" grp

Reply:
done